src/main/java/org/opensearch/security/dlic/rest/api/RestApiPrivilegesEvaluator.java

```diff
@@ -446,7 +448,11 @@ private String checkAdminCertBasedAccessPermissions(RestRequest request) throws
         }
 
         // Certificate based access, Check if we have an admin TLS certificate
-        SSLRequestHelper.SSLInfo sslInfo = SSLRequestHelper.getSSLInfo(settings, configPath, request, principalExtractor);
+        final SecurityRequestChannel securityRequest = SecurityRequestFactory.from(
+            request,
+            /* OK to be null so long as a response isn't sent */ null
```

Comment:
Can you elaborate on this comment?

Reply:
Thanks for catching, I created SecurityRequest & SecurityRequestChannel to handle this kind of case - I just need to clean this up to use SecurityRequest.

Reply:
Just to be clear - this confusing comment and the null value have been removed.